apps/pie-microsite/package.json

```diff
@@ -1,28 +1,26 @@
 {
   "name": "pie-microsite",
   "description": "todo",
-  "version": "1.17.0",
+  "version": "1.18.0",
   "main": "index.js",
   "keywords": [],
   "author": "JustEatTakeaway - Vue Design System Team",
   "license": "MIT",
-  "packageManager": "yarn@3.2.1",
```

Comment:
Is this now inherited from the root?

Reply:
Unsure - this shouldn't be removed so going to add back in now. Good spot!